Podfile

```diff
@@ -2,9 +2,11 @@ platform :ios, '9.0'
 # ignore all warnings from all pods
 inhibit_all_warnings!
 target 'ooniprobe' do
-    pod 'measurement_kit',
-        :git => 'https://github.com/measurement-kit/measurement-kit.git',
-	:tag => 'v0.8.3'
+  pod 'measurement_kit',
+    :git => 'https://github.com/measurement-kit/ios-libs.git'
```

Comment:
I should probably add a tag here, but for now I guess we can avoid it, right?

Reply:
for now we can avoit it, later better use tags

Reply:
agreed!